### PR TITLE
Removed stackoverflow from 'usage' template

### PR DESCRIPTION
**Pull request by Hilbrand**
Stack Overflow is for a specific target group and for source code related questions: https://stackoverflow.com/help/on-topic So I would not advice people to visit stack overflow for 'usage' questions.

### CODE REVIEW

**Comment by davidgraeff** [comment]
I don't like the culture and sometimes hostile atmosphere in the community board towards developers. So I think we should advertise stackoverflow more, for binding development questions. But you are right that the wording is not good enough and doesn't differentiate between "support" and "development question".

**Comment by hakan42** [comment]
Maybe reword it a bit so its is clearer that the openHAB forums are not specifically geared towards generic Linux questions?

**Comment by mhilbush** [comment]
Having developed a few bindings, I would think it's important to differentiate between help for general Java questions/approaches versus help with the framework. Stackoverflow is an excellent resource for Java questions (someday maybe I won't need to use it so much 😄). However, with the move of ESH into the openHAB core, and with the ESH forum no longer available for framework-specific questions, the community board is the only place to get that kind of help.

Would it therefore make sense to mention both resources in this template?

**Comment by davidgraeff** [comment]
> Would it therefore make sense to mention both resources in this template?

We can just start to use stackoverflow. At the moment no developer has subscribed to the "openhab" tag as far as I know.

**Comment by mhilbush** [comment]
Sorry, I think I misunderstood the purpose of the **usage** template. I realize it may focused on usage of the openHAB system (versus usage of the framework).

**Comment by openhab-5iver** [comment]
This PR makes sense to me, but @davidgraeff just added very much needed issue templates and included Stackoverflow in them #4933. Why would we dilute the community knowledgebase by directing people _away_ from the forum?!

**Comment by davidgraeff** [comment]
See https://community.openhab.org/t/openhab-subreddit-locked/67939?u=david_graeff

It doesn't hurt to spread across multiple platforms, so that other communities can see that we exist. and Stackoverflow is made for developers and actively used for a lot of communities, that's why I suggested it.

**Comment by wborn** [comment]
It would certainly be nice to also have a bigger presence at the other platforms. But it is also frustrating to be directed to a place where fewer answers to questions are provided. This is of course a chicken/egg problem.
